### ISSUE TITLE
Translation to Brazilian portuguese (pt_br)

### ISSUE THREAD

**Issue by danielramosbh74** [evbacher/gd2md-html#65]
Hi @evbacher, I'd like to help you starting translating your G Suite Add On to Brazilian portuguese.

I'm very impressed with your work, because you have the rare hability to simplify things (ex. your Private Policy (very short and objective) and Terms of Service (just linking to the conventional Apache Licence).

I've also read your resume and think I've finally "find my way", because I like to do the same thing you're doing. 
I love to organize, simplify and resume information, and that's one of the reasons I'd like to help you translating your work to brazilian portuguese.

Excuse me for this long message, but I was searching for an add on like yours for a long time and now I've found it.

**Comment by kamikazebr** [comment]
I want to help with it too!!!

It's addon it's amazing! And i'm feeling so much grateful by found it. 

Help a lot in one big work and that work helped alot peoples!

So, how can start work on it? 

Can i fork and start the translate?

Do you know how @danielramosbh74 ?

**Comment by danielramosbh74** [comment]
Yes @kamikazebr, it was what I've done, I've forked the project..

I've also created 2 new folders: /translations/pt-BR
But I'm not sure if @evbacher already commited it.
If he had commited it you can go on translating starting were I've stoped.

**Comment by evbacher** [comment]
I just committed your pull request, @danielramosbh74  -- thanks! @kamikazebr, feel free to contribute as well -- glad you like this addon.

I may not get to these things quickly, but if I cannot review soon, I'll let you know.

**Comment by kamikazebr** [comment]
Nice danielramosbh74, ty!

I just realized now that I also want to contribute with the translation of the addon, for example: the panel inside the GDocs.

So, looking at the source code, I believe it is necessary to adapt to i18n.

I found this out in my research

https://github.com/i18next/i18next

I think it would be easy to introduce and create an internationalization file, even for future translations.

What do you think of this?
